compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScene.skiko.kt

```diff
@@ -238,8 +240,8 @@ interface ComposeScene {
      *
      * @see InteropViewCatchPointerModifier
      */
-    @Deprecated("To be removed. Temporary hack for iOS interop")
-    fun hitTestInteropView(position: Offset): Boolean
+    @ExperimentalComposeUiApi
```

Comment:
It shouldn't be marked as `@ExperimentalComposeUiApi`

Reply:
Done